### DISCUSSION TITLE
Weekly Update (July 30 -> Aug 11)

### DISCUSSION THREAD

**Discussion by mayrinck** [mayrinck/FOSSonMicrosoftStore#32] [Announcements]
Hey there guys, and I'm sorry about the weekly updates becoming delayed, but right now I'm being a bit busy with other things in life, still, the project is having its progress over time...

We have the following news right now:

- We are trying to push GIMP back to the store after giving some more permissions to the package (hope this solves the problem). 
- Antimicro has been downloaded **359 times** since launch.
- Dia Diagram Editor has been downloaded **107 times** since launch.

**Comment by JuliaAgostino** [comment]
> Hey there guys, and I'm sorry about the weekly updates becoming delayed, but right now I'm being a bit busy with other things in life, still, the project is having its progress over time...
> 
> We have the following news right now:
> 
> * We are trying to push GIMP back to the store after giving some more permissions to the package (hope this solves the problem).
> * Antimicro has been downloaded **359 times** since launch.
> * Dia Diagram Editor has been downloaded **107 times** since launch.

Okay, that's great news!
